sets/treeset/iterator.go

```diff
@@ -6,7 +6,7 @@ package treeset
 
 import (
 	"github.com/emirpasic/gods/containers"
-	rbt "github.com/emirpasic/gods/trees/redblacktree"
+	"github.com/spewspews/gods/trees"
```

Comment:
I will change this to reference this repo as to keep the zero-dependencies policy if that's OK